homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -321,6 +337,56 @@
           "description": "In which work area the mower should mow."
         }
       }
+    },
+    "set_schedule": {
+      "name": "Set schedule",
+      "description": "Overwrite, add or remove a schedule. For mowers with work areas: If a work area is selected, than the changes only have effect on this work area. E.g. If you overwrite all schedules in work area `0`, than the schedules in work area `123456` are not affected.",
+      "fields": {
+        "mode": {
+          "name": "Mode",
+          "description": "Selects if the current schedule should be overwritten, expanded or if one of the schedules schould be removed."
+        },
+        "start": {
+          "name": "Start",
+          "description": "Start of the schedule, seconds will be ignored."
+        },
+        "end": {
+          "name": "End",
+          "description": "End of the schedule, seconds will be ignored."
+        },
+        "monday": {
+          "name": "Monday",
+          "description": "If set, then the schedule will be activated on this day."
+        },
+        "tuesday": {
+          "name": "Tuesday",
+          "description": "[%key:component::husqvarna_automower::services::set_schedule::fields::monday::description%]"
+        },
+        "wednesday": {
+          "name": "Wednesday",
+          "description": "[%key:component::husqvarna_automower::services::set_schedule::fields::monday::description%]"
+        },
+        "thursday": {
+          "name": "Thursday",
+          "description": "[%key:component::husqvarna_automower::services::set_schedule::fields::monday::description%]"
+        },
+        "friday": {
+          "name": "Friday",
+          "description": "[%key:component::husqvarna_automower::services::set_schedule::fields::monday::description%]"
+        },
+        "saturday": {
+          "name": "Saturday",
+          "description": "[%key:component::husqvarna_automower::services::set_schedule::fields::monday::description%]"
+        },
+        "sunday": {
+          "name": "Sunday",
```

Comment:
I think the days of the week are in the common strings

Reply:
done in https://github.com/home-assistant/core/pull/126311/commits/ad53872ed11267854288be42f16b8138ebd90146